index.html

```diff
@@ -4,7 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
     <title>Personal Website</title>
   </head>
-  <body></body>
+
+  <body>
+    <header>
+      <h1>Adriana Morales Tremolada</h1>
+      <h2>Human</h2>
+      <nav>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>About me</h2>
+      </section>
+      <section>
+        <h2>Interests</h2>
+      </section>
+      <section>
+        <h2>What the future holds</h2>
+      </section>
+      <section>
+        <h2>Contact</h2>
+      </section>
+    </main>
+
+    <footer>
+      <div>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </div>
+      <div>
+        <h4>Let's connect:</h4>
```

Comment:
Don't skip headlines. Read more about here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes
If you nees smaller headline, you would style it with css. But semantically a `h2` element would be correct here.

Reply:
Noted!🙂